### DISCUSSION TITLE
ZeroBot插件 出现握手错误怎么解决？websocket: bad handshake

### DISCUSSION THREAD

**Discussion by fppgga** [FloatTech/ZeroBot-Plugin#323] [Q&A] [answered]
![2e55809d8871ca3c720a3be07b2aff3](https://user-images.githubusercontent.com/78149460/179411457-813bf1b0-cd37-4a8e-ae13-51e60fd09f0b.jpg)

**Comment by fumiama** [comment, marked as answer]
配错了

**Comment by shinkisunday** [comment]
碰到相同问题了想问下怎么解决的
